taxcalc/functions.py

```diff
@@ -1008,19 +1009,18 @@ def AMTI(c60000, _exact, e60290, _posagi, e07300, x60260, c24517, e37717,
 
     _line46 = min(_alminc, c62720)
 
-    _line48 = min(_alminc, c62720) - min(_line45, _line46)
+    _line47 = min(_line45, _line46)
 
-    _amt15pc = min(_line48, max(0., AMT_CG_thd2[MARS - 1] - c24520 + _line45))
+    _line48 = min(_alminc, c62720) - _line47
 
-    if c04800 == 0:
-        _amt15pc = max(0., _line46 - AMT_CG_thd1[MARS - 1])
+    _amt15pc = min(_line48, max(0., AMT_CG_thd2[MARS - 1] - c24520 - _line45))
 
-    if _ngamty != _amt15pc:
-        _amt20pc = _line46 - _amt15pc - min(_line45, _line46)
+    if _ngamty != (_amt15pc + _line47):
+        _amt20pc = _line46 - _amt15pc - _line47
     else:
         _amt20pc = 0.
 
-    if c62730 != 0:
+    if c62720 != 0:
```

Comment:
Correspond to the comment "If line 38 is zero or blank, skip lines 59 through 61 and go to line 62. Otherwise, go to line 59. " on [form 6251](https://www.irs.gov/pub/irs-pdf/f6251.pdf). It seems like `c62730` stands for line 39 rather than 38.